src/mscorlib/src/System/Threading/WaitHandle.cs

```diff
@@ -448,7 +448,7 @@ public static int WaitAny(WaitHandle[] waitHandles, int millisecondsTimeout)
                                         WaitHandle toWaitOn)
         {
 #if PLATFORM_UNIX
-            throw new PlatformNotSupportedException();
+            throw new PlatformNotSupportedException(SR.Arg_PlatformNotSupported); // https://github.com/dotnet/coreclr/issues/10441
```

Comment:
> I'm more concerned with the 500-1000 PNSE in CoreFX 

Of course.  I'm just wondering why we'd go to the trouble of adding this specific error messages for some cases but not others.  To me, the most important place to have specific error messages is when a) it's not immediately obvious what caused it (e.g. the method is generally supported but not if a certain argument or certain combination of arguments is used) or b) there's an alternative we can recommend.  For everything else, I think the default message is fine, and that we shouldn't be spending time or space on resources adding in specific messages.  I also like consistency, though.